std_msgs/Header header
sensor_msgs/Image left_eye_frame
sensor_msgs/Image right_eye_frame